### PR TITLE
Initialize AppVeyor CI to test plugin on Windows

### PR DESCRIPTION
**Pull request by ashmaroli**
- [x] Set up AppVeyor
- [x] Update README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 Rebuild your Jekyll site when a file changes with the `--watch` switch.
 
-[![Build Status](https://travis-ci.org/jekyll/jekyll-watch.svg?branch=master)](https://travis-ci.org/jekyll/jekyll-watch)
+[![Linux Build Status](https://travis-ci.org/jekyll/jekyll-watch.svg?branch=master)][travis]
```
Comment (XhmikosR) [inline README.md]: I'd go with shields.io here too because it'll be faster and consistent. Also, I'd add `/master` here too.

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  only:
+    - master
+
+build: off
+
+install:
+  - SET PATH=C:\Ruby%RUBY_FOLDER_VER%\bin;%PATH%
+  - bundle install --retry 5 --jobs=%NUMBER_OF_PROCESSORS% --clean --path vendor\bundle
+
+environment:
```
Comment (XhmikosR) [inline appveyor.yml]: I'd move the environment block after the branches.

**Comment by XhmikosR** [comment]
BTW, shouldn't this fail since #76 isn't merged yet?

**Comment by ashmaroli** [comment]
> shouldn't this fail...

The scenario described in #76 isn't being tested.

**Comment by ashmaroli** [comment]
@XhmikosR Thank you for reviewing. I'll update shortly..

**Comment by XhmikosR** [comment]
I made https://github.com/jekyll/jekyll/pull/7312 BTW, probably you based this one on the upstream file so you might wait for that.

**Comment by DirtyF** [comment]
@jekyllbot: merge +dev

**Pull request merged**